pairing/src/bls12_381/ec.rs

```diff
@@ -18,7 +18,7 @@ macro_rules! curve_impl {
         }
 
         impl ::std::fmt::Display for $affine {
-            fn fmt(&self, f: &mut ::std::fmt::Formatter) -> ::std::fmt::Result {
+            fn fmt(&self, f: &mut ::std::fmt::Formatter<'_>) -> ::std::fmt::Result {
```

Comment:
Note: In pairing I did a find and replace on `::std::fmt::Formatter`/`::std::fmt::Formatter<'_>`. `cargo fix --edition-idioms` would fail here and try to double add the `<'_>`.